meson.build

```diff
@@ -3052,6 +3053,7 @@ summary({
         'default user $PATH'                        : default_user_path != '' ? default_user_path : '(same as system services)',
         'systemd service watchdog'                  : service_watchdog == '' ? 'disabled' : service_watchdog,
         'time epoch'                                : f'@time_epoch@ (@alt_time_epoch@)',
+        'TPM2 nvpcr base'                           : run_command(sh, '-c', 'printf 0x%x @0@'.format(get_option('tpm2-nvpcr-base')), check : true).stdout()
```

Comment:
Any chance the formatting could be done natively in meson, rather than shelling out? Accumulated shell-outs are one of the things that make meson configure runs slow

Reply:
Filed an RFE: https://github.com/mesonbuild/meson/issues/15201